src/js/reducers/datasetsReducer.ts

```diff
@@ -45,17 +44,18 @@ export function datasetsReducer(state: DatasetState, action: ActionType<typeof d
     const transforms: Transforms[] = state.getIn([String(id), 'transform']) || List();
 
     if (transform.type === 'formula') {
-      state = state.setIn([String(id), '_schema', transform.as],
+      state = state.setIn(
+        [String(id), '_schema', transform.as],
         Column({
           name: transform.as,
           type: 'number',
-          mtype: MTYPES.QUANTITATIVE,
+          mtype: 'quantitative',
```

Comment:
perhaps a TODO to make an enum for measure types?

Reply:
Turns out this exists in vega-lite types, will import.